Makefile

```diff
@@ -13,6 +13,13 @@ imports = \
 xcodeproj:
 	swift package generate-xcodeproj
 
+bootstrap: common-crypto-mm xcodeproj
+
+common-crypto-mm:
+	-@mkdir "$(COMMON_CRYPTO_PATH)" || true
```

Comment:
`mkdir -p` should negate the need for `|| true`

Reply:
lol very true